src/Tizen.Wearable.CircularUI.Forms.Renderer/PopupEntryRenderer.cs

```diff
@@ -248,6 +254,17 @@ void UpdateIsPopupOpened()
                 HidePopup();
             }
         }
+
+
+        protected override void UpdateTextColor()
```

Comment:
Before, We do not apply TextColor on PopupEntry, If you should apply TextColor on PopupEntry, we also change `CreatePopup` method to change _editor.TextColor

Reply:
TextColor set in ShowPopup().

            _editor.TextStyle = Control.TextStyle;
            _editorPopup.Color = _popupBackgroundColor;